src/main/cpp/src/row_conversion.cu

```diff
@@ -2325,6 +2334,9 @@ std::unique_ptr<table> convert_from_rows_fixed_width_optimized(
         num_rows, num_columns, size_per_row, dev_column_start.data(), dev_column_size.data(),
         dev_output_data.data(), dev_output_nm.data(), child.data<int8_t>());
 
+    for (auto &col : output_columns) {
+      col->set_null_count(cudf::null_count(col->view().null_mask(), 0, col->size()));
+    }
```

Comment:
This is similar.

I'm also not sure if we can compute null mask and null count in the kernel above  this.

Reply:
Based on prior discussion with @hyperbolic2346, it might  well be possible to track the null-mask and null-count in the kernel. I think it would be better not to change that kernel till after 23.06. (Unless @hyperbolic2346 would like to have a go at it? :])

Reply:
I've documented why this is being set externally, and added a to-do to move this to the kernel if possible.